packages/sdk-demo/src/types/index.d.ts

```diff
@@ -0,0 +1,4 @@
+export interface ThemeConfig {
+  themeIndex: 'material' | 'monite';
```

Comment:
I would like to name props according to [MUI naming convention](https://mui.com/material-ui/customization/dark-mode/#dark-mode-by-default):
```
themeIndex => variant | palette
colorMode   => mode
```

Reply:
fixed. I chose 'variant' because 'palette' is just a part of the theme options object. 'variant' is more general